scripts/libraries/ml/ml/postprocessing.py

```diff
@@ -33,6 +33,9 @@
 from ulab import numpy as np
 
 
+_NO_DETECTION = const(())
```

Comment:
Had to be here otherwise I got a compile error: SyntaxError: can't assign to expression